cvat/apps/documentation/user_guide.md

```diff
@@ -1,232 +1,442 @@
 
     ``input_type``: the following input types are available ``select``, ``checkbox``, ``radio``, ``number``, ``text``.
 
-    ``attribute_name``: for example *age, quality, parked*
+    ``attribute_name``: for example, _age, quality, parked_
 
-    ``attribute_value``: for example *middle-age, good, true*
+    ``attribute_value``: for example, _middle-age, good, true_
 
     Default value for an attribute is the first value after "``:``".
 
-    For ``select`` and ``radio`` input types the special value is available: ``__undefined__``. Specify this value first if an attribute should be annotated explicitly.
+    For ``select`` and ``radio`` input types the special value is available: ``__undefined__``.
+    Specify this value first if an attribute should be annotated explicitly.
+
+    **Bug Tracker**. Specify full bug tracker's URL if you have it.
+
+    **Dataset Repository**.  URL link of the repository optionally specifies the path to the repository for storage
+    (``default: annotation / <dump_file_name> .zip``).
+    The .zip and .xml file extension of annotation are supported.
+    Field format: ``URL [PATH]`` example: ``https://gitlab-icv.inn.intel.com/project/repos.git  [1/2/3/4/annotation.xml]``
+
+    Supported URL formats :
+    - ``https://gitlab-icv.inn.intel.com/project/repos[.git]``
+    - ``gitlab-icv.inn.intel.com/project/repos[.git]``
+    - ``git@gitlab-icv.inn.intel.com:project/repos[.git]``
+
+    The task will be highlighted in red after creation if annotation isn't synchronized with the repository.
+
+    **Use LFS**. If the annotation file is large, you can create a repository with
+    [LFS](https://git-lfs.github.com/) support.
+
+    **Source**. To create huge tasks please use ``shared`` server directory (choose ``Share`` option in the dialog).
+
+    **Z-Order**. Defines the order on drawn polygons. Check the box for enable layered displaying.
+
+    **Overlap Size**. Use this option to make overlapped segments.
+    The option makes tracks continuous from one segment into another.
+    Use it for interpolation mode. There are several options for using the parameter:
+    - For an interpolation task (video sequence).
+    If you annotate a bounding box on two adjusted segments they will be merged into one bounding box.
+    If overlap equals to zero or annotation is poor on adjacent segments inside a dumped annotation file,
+    you will have several tracks, one for each segment, which corresponds to the object.
+    - For an annotation task (independent images).
+    If an object exists on overlapped segments, the overlap is greater than zero
+    and the annotation is good enough on adjacent segments, it will be automatically merged into one object.
+    If overlap equals to zero or annotation is poor on adjacent segments inside a dumped annotation file,
+    you will have several bounding boxes for the same object.
```

Comment:
If overlap equals to zero we don't have several bounding boxes for the same object because we don't have same objects